src/main/java/uk/co/evoco/webdriver/configuration/driver/ConfiguredChromeDriver.java

```diff
@@ -38,6 +43,32 @@ public WebDriver getLocalDriver() throws IOException {
      */
     public ChromeOptions getOptions() {
         ChromeOptions chromeOptions = new ChromeOptions();
+        Map<String, Object> chromePrefs = new HashMap<>();
+        Iterator<Map.Entry<String, JsonNode>> browserPreferences = TestConfigHelper.get()
+                .getBrowserPreferences(BrowserType.CHROME)
+                .fields();
+        while (browserPreferences.hasNext()) {
+            Map.Entry<String, JsonNode> entry = browserPreferences.next();
+            JsonNode value = entry.getValue();
+            String key = entry.getKey();
+            switch (value.getNodeType()) {
+                case BOOLEAN:
+                    chromePrefs.put(key, value.asBoolean());
+                    break;
+                case NUMBER:
+                    chromePrefs.put(key, value.asInt());
+                    break;
+                default:
```

Comment:
I'm not sure about this.  We have the config where a user might not be clear that they are setting a directory that will be relative to their 'user.home' directory with no way to override it?  I think we give file paths relative to where the configuration is loaded from or we give fully qualified file paths.  Can you rethink?

Reply:
changed to create the down load folder in target folder and added init tests

Reply:
I don't think using target folder is a solution to this.  Look at the next comment!